packages/types/src/base.ts

```diff
@@ -2,12 +2,15 @@ import type { AccountId, AssetId, ChainId } from '@shapeshiftoss/caip'
 
 import type { PartialRecord } from './utility'
 
-export type BIP44Params = {
+export type DefaultBIP44Params = {
```

Comment:
💜

Reply:
suggestion: can we call this `BaseBIP44Params` @kaladinlight ?
I get the intent of *not* passing change/address_index when we're actually not incrementing these (or rather, not yet for address_index! inb4 custom derivation work), but in and by itself, it's an incomplete BIP44 path, which suggests base terminology would be a better fit

Reply:
I had a similar thought when updating. Might as well make the change now as long as we are in here!

Reply:
9e079dcac871fc638e895b9e82a05d2425c8d1bb